WpfSendKeys/SendKeysParser.cs

```diff
@@ -84,7 +33,10 @@ public IEnumerable<KeyPressInfo> ParseSendKeysString(string text)
                 }
                 else
                 {
-                    Error("Unexpected character: '" + CurrentChar + "'");
+					//TODO: HandleNewline here
+					ParseChar();
+
+                    //Error("Unexpected character: '" + CurrentChar + "'");
```

Comment:
The idea being I don't want to have to peek ahead and consume `\r\n` together. I want to ignore `\r` completely and introduce `Key.Enter` when we encounter `\n`.